assets/js/contact_us_form_element.ts

```diff
@@ -0,0 +1,47 @@
+class ContactUsFormElement extends HTMLElement {
+  connectedCallback() {
+    if (this.isInMaintenanceWindow) {
+      this.hide();
+    }
+  }
+
+  get alert(): HTMLElement | null {
```

Comment:
Should probably qualify this as the "maintenance" alert, for consistency with the rest of the implementation treating maintenance window as a feature of the contact form.

```suggestion
  get maintenanceAlert(): HTMLElement | null {
```

Reply:
Also would any code scanning tools confused this for `window.alert` possibly?

Reply:
> Also would any code scanning tools confused this for `window.alert` possibly?

It's possible, at least especially if ever this would be destructured:

```
const { alert } = this;
```

Shouldn't be an issue if I rename it though.

Reply:
Renamed to `maintenanceAlert` in e660bb84.